testsuite/MDAnalysisTests/analysis/test_leaflet.py

```diff
@@ -42,3 +44,11 @@ def test_leaflet_finder(self):
         assert_equal(bottom_heads.indices, np.arange(2521,4670,12), err_msg="Found wrong leaflet lipids")
 
 
+    def test_string_vs_atomgroup_proper(self):
+        from MDAnalysis.analysis.leaflet import LeafletFinder
```

Comment:
Thanks for explaining. I think it's fair to say that it's always worthwhile to rethink existing code, especially if one runs into more ancient pieces.